routes/api/bdgtRoutes.js

```diff
@@ -1,5 +1,21 @@
+const Purchase = require("../../lib/purchase");
+const User = require("../../lib/User");
 const router = require("express").Router();
 
-// TODO: Add BDGT API ROUTES 
+// TODO: Add BDGT API ROUTES
+router.post("/purchase", (req, res) => {
+  Purchase.create({
+    price: req.body.price,
+  });
 
-module.exports = router;
\ No newline at end of file
+  if (req.session.user) {
+    savePurchase(req.body.purchase, req.session.user.id);
+    res.send("Purchase Saved!");
+  } else {
+    res.status(401).json({ message: "Unable to save purchase" });
+    return;
+  }
+});
+router.post("/income", (req, res) => {});
```

Comment:
we can create 1 API to handle these and change the type, rather than creating multiple API's and having to code out more